azure/template.json

```
@@ -60,63 +46,65 @@
                 "description": "An array of IPs that are permitted to access the app service."
             }
         },
-        "keyVaultCertificateName": {
+        "certificateName": {
```

```suggestion
        "keyvaultCertificateName": {
```
to directly match the variable name in the Wildcard Cert Variables v2 variable group

---

Updated